docker/Dockerfile

```diff
@@ -25,7 +32,7 @@ RUN sed -i.bak 's/^\(port.https=\).*/\1443/'              conf/config.properties
 RUN sed -i.bak 's/^\(upgradeInterval=\).*/\186400000000/' conf/config.properties
 
 # hack until loklak support no-daemon
-RUN echo "while true; do sleep 10;done" >> bin/start.sh
+COPY docker-cmd.sh docker-cmd.sh
 
 # start loklak
-CMD ["bin/start.sh"]
+CMD ["./docker-cmd.sh"]
```

Comment:
the shell file is located at `docker/docker-cmd.sh` , should this line be updated accordingly ?